docs/basic/params.rst

```diff
@@ -143,8 +143,8 @@ If the variables containing the data to send to the database come from an
 untrusted source (such as data coming from a form on a web site) an attacker
 could easily craft a malformed string, either gaining access to unauthorized
 data or performing destructive operations on the database. This form of attack
-is called `SQL injection`_ and is known to be one of the most widespread forms
-of attack to database systems. Before continuing, please print `this page`__
+is called an `SQL injection`_ and is known to be one of the most widespread forms
```

Comment:
I  don't think inserting `an` is an improvement here.
(No opinion either way on `to` vs `on` on the line below.)